presto-main/src/test/java/com/facebook/presto/type/TestBigintOperators.java

```diff
@@ -256,6 +257,14 @@ public void testOverflowDivide()
         assertInvalidFunction(format("%s / -1", Long.MIN_VALUE), NUMERIC_VALUE_OUT_OF_RANGE);
     }
 
+    @Test
+    public void testIndeterminate()
+    {
+        assertOperator(INDETERMINATE, "cast(1 as bigint)", BOOLEAN, false);
+        assertOperator(INDETERMINATE, "cast(4499999999 as bigint)", BOOLEAN, false);
+        assertOperator(INDETERMINATE, "4499999999", BOOLEAN, false);
```

Comment:
The operator does not, but it should be possible to "invoke" it (the engine should short-circuit and return true if the value is null). The point is that you should be able to write a test and make sure it does the right thing.